app/src/services/mongo/index.ts

```diff
@@ -1,15 +1,21 @@
-import {ContainerModule} from 'inversify';
-import ServiceInterface, {SERVICE} from '../../interfaces/ServiceInterface';
+import { ContainerModule } from 'inversify';
+import ServiceInterface, { SERVICE } from '../../interfaces/ServiceInterface';
```

Comment:
Revert spacing changes - check your linter, it might not be working :(

Reply:
Srry about that, I had it turned off as I couldn't differentiate between actual problems and eslint whining